hooks/pre-command

```diff
@@ -10,8 +10,8 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 GH_TOKEN="${GH_TOKEN:-${GITHUB_TOKEN:-}}"
 OBLT_CLI_USERNAME="${BUILDKITE_PLUGIN_OBLT_CLI_USERNAME:-obltmachine}"
 OBLT_CLI_SLACK_CHANNEL="${BUILDKITE_PLUGIN_OBLT_CLI_SLACK_CHANNEL:-#observablt-bots}"
-OBLT_CLI_VERSION="${BUILDKITE_PLUGIN_OBLT_CLI_VERSION:-7.3.0}"
-OBLT_CLI_VERSION_FILE="${BUILDKITE_PLUGIN_OBLT_CLI_VERSION_FILE:-}"
+OBLT_CLI_VERSION="${BUILDKITE_PLUGIN_OBLT_CLI_VERSION:-}"
+OBLT_CLI_VERSION_FILE="${BUILDKITE_PLUGIN_OBLT_CLI_VERSION_FILE:-"${DIR}/../.default-oblt-cli-version"}"
```

Comment:
I'm a bit confused, what's the reason for using this path rather than the hardcoded version in the previous line.

Will this path be relative to the plugin regardless or the consumers?

Reply:
This path is relative to the plugin.

The reason is how the `get_version_from_input_or_file` function determines which to use.

The pseudo code is something like this:

```
if input_version is not empty
then version = input_version
else check files
```

Hence, when the default value is always not empty, it will always take the input_version.

Reply:
Another reason why I chose to use the file is, that it will make it easier to update it with updatecli.

Reply:
IIUC, the plugin without a specified version, will use the version in the `. default-oblt-cli-version`, right? In other words, how do I configure the plugin for the consumer to use the latest version?

Shall we always think about using `updatecli` to update the version when a new release is out? If so, how do you envision aligning both releases, the one for `oblt-cli` and the one for this plugin?

Reply:
You're right; it's not possible to use the latest version.

To make it more straightforward, I would personally introduce something like a pseudo version called `latest`.

e.g.

```
plugins:
  - elastic/oblt-cli#v1.0.0:
       version: latest
```

From my understanding it's the same in oblt-actions since https://github.com/elastic/oblt-actions/pull/54

if you don't provide a version the default will be used.

Reply:
`       version: latest` <--- I like that approach

Reply:
Alright, I will add it in a follow-up.. also in oblt-actions